spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -96,15 +96,35 @@
 
   %i[above below left_of right_of near count minimum maximum between text id
      class style visible obscured exact exact_text normalize_ws match wait
-     filter_set focused href alt title download].each do |attr|
-    it 'registers an offense for abstract matcher when ' \
+     filter_set focused alt title download].each do |attr|
+    it 'does not register an offense for abstract matcher when ' \
        "first argument is element with replaceable attributes #{attr} " \
-       'for `have_link`' do
-      expect_offense(<<-RUBY, attr: attr)
+       'for `have_link` without `href`' do
+      expect_no_offenses(<<-RUBY, attr: attr)
         expect(page).to have_css("a[#{attr}=foo]")
-                        ^^^^^^^^^^^^^{attr}^^^^^^^ Prefer `have_link` over `have_css`.
         expect(page).to have_css("a[#{attr}]")
-                        ^^^^^^^^^^^^^{attr}^^^ Prefer `have_link` over `have_css`.
+      RUBY
+    end
+
+    it 'registers an offense for abstract matcher when ' \
```

Comment:
For spec completeness we need a[href=] with no other attributes

Reply:
Indeed it is. I have added a test: https://github.com/rubocop/rubocop-rspec/blob/906aef1e2e6d1a13379a8326b6d63c3e7dd79594/spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb#L132-L141